### PR TITLE
Introduced Style CI

### PR DESCRIPTION
**Pull request by dantleech**

### CODE REVIEW

File: Admin/BlogAdmin.php [hunk 1]
```diff
@@ -34,9 +31,9 @@ protected function configureFormFields(FormMapper $mapper)
     {
         $mapper->add('name', 'text');
         $mapper->add('parent', 'doctrine_phpcr_odm_tree', array(
-            'root_node' => $this->blogRoot, 
-            'choice_list' => array(), 
-            'select_root_node' => true)
+            'root_node' => $this->blogRoot,
+            'choice_list' => array(),
+            'select_root_node' => true, )
```
Comment (dbu) [inline Admin/BlogAdmin.php]: this is weird. the parenthesis should be on the next line

File: Block/TagCloudBlock.php [hunk 2]
```diff
@@ -17,16 +16,15 @@
...
 use Symfony\Cmf\Bundle\BlogBundle\Repository\TagRepository;
 
 /**
- * Tag Cloud block service
+ * Tag Cloud block service.
  *
  * @author Daniel Leech <daniel@dantleech.com>
  */
-class TagCloudBlockService extends BaseBlockService
+class TagCloudBlock extends BaseBlockService
```
Comment (dbu) [inline Block/TagCloudBlock.php]: erm, we should check if we need to fix the file name or the class name. not sure how this ever could work.

File: Tests/Unit/Util/PostsUtilTest.php [hunk 3]
```diff
@@ -22,7 +21,7 @@ public function dataProviderSlugify()
             array('this is a slug', 'this-is-a-slug'),
             array('This Is A SLUG', 'this-is-a-slug'),
             array('je suis une dévelopeur web', 'je-suis-une-developeur-web'),
-            array('foo!"  812391 !"£ %!"$£*%!"$£', 'foo-812391'),
+            array('foo!"  812391 !"£ %!"$£*%!"$£', 'foo-812391-lb-lblb'),
```
Comment (dbu) [inline Tests/Unit/Util/PostsUtilTest.php]: what?

Comment (lsmith77) [inline Tests/Unit/Util/PostsUtilTest.php]: i did this change .. I suspect that `` !"£ %!"$£*%!"$£`` was previously ignored and stripped by the slugifyer and is now transliterated to.

File: Block/TagCloudBlock.php (hunk 2 above)
Comment (lsmith77) [inline Block/TagCloudBlock.php]: I think this entire class is pretty broken as ``TagRepository`` also doesn't exist .. it also appears like the class isn't even configured as a service either.

**Comment by lsmith77** [comment]
Hmm .. there is an issue with the slugifier tests .. locally they are passing with PHP 5.6.14

```
......FFF
Time: 15.57 seconds, Memory: 62.75Mb
There were 3 failures:
1) Symfony\Cmf\Bundle\BlogBundle\Tests\Unit\Util\PostsUtilTest::testSlugify with data set #2 ('je suis une dévelopeur web', 'je-suis-une-developeur-web')
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'je-suis-une-dvelopeur-web'
+'je-suis-une-developeur-web'
/home/travis/build/symfony-cmf/BlogBundle/Tests/Unit/Util/PostsUtilTest.php:35
2) Symfony\Cmf\Bundle\BlogBundle\Tests\Unit\Util\PostsUtilTest::testSlugify with data set #3 ('foo!"  812391 !"£ %!"$£*%!"$£', 'foo-812391-lb-lblb')
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'foo-812391'
+'foo-812391-lb-lblb'
/home/travis/build/symfony-cmf/BlogBundle/Tests/Unit/Util/PostsUtilTest.php:35
3) Symfony\Cmf\Bundle\BlogBundle\Tests\Unit\Util\PostsUtilTest::testSlugify with data set #4 ('ĉràzŷ& cĥärs', 'crazy-chars')
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'rz-crs'
+'crazy-chars'
```

File: Block/TagCloudBlock.php (hunk 2 above)
Comment (dbu) [inline Block/TagCloudBlock.php]: so lets delete the class? or leave this alone and try to wrap up #59? or just deprecate and stop this bundle completely to focus on things we actually maintain?

**Comment by dantleech** [comment]
Personally I would be up for deprecating this whole bundle..

**Comment by lsmith77** [comment]
well we never had a stable release ..

**Comment by dbu** [comment]
> well we never had a stable release ..

does this mean you agree that we should deprecate it? or do you think we
should have a release first? i think the later would not be worth the
effort.

**Comment by lsmith77** [comment]
I agree that we can deprecate it ..

**Comment by dbu** [comment]
we abandoned this bundle now

**Pull request closed without merging**